sites/example-project/src/pages/links-to-page-headers/+page.md

```diff
@@ -0,0 +1,4 @@
+
+Doesn't currently work
+
+[Deep in the text](/text-and-metrics/text#more-nostra-non-parsque-talia-discussit)
```

Comment:
Could we open an issue instead of doing this?

Reply:
good idea, will do upon merge

Reply:
#1871